controller/Publish.php

```diff
@@ -62,4 +64,43 @@ public function wizard()
             $this->setView('DeliveryMgmt/wizard_error.tpl');
         }
     }
+
+    public function publish()
+    {
+        $this->defaultData();
+
+        $formContainer = new PublishForm(['class' => $this->getCurrentClass()]);
+        $myForm = $formContainer->getForm();
```

Comment:
Maybe call it just `form`? `$myForm` sounds weird to me

If the suggestion to change it is accepted, please change it here and wherever it is used